packages/neon-nep5/__integration__/main.ts

```diff
@@ -97,3 +102,62 @@ describe("getToken", () => {
     expect(balanceNum).toBeGreaterThan(0);
   });
 });
+
+describe("getTokens", () => {
+  test("without balance", async () => {
+    const info = await getTokens(TESTNET_URL, [
+      CONST.CONTRACTS.TEST_RPX,
+      CONST.CONTRACTS.TEST_LWTF,
+      CONST.CONTRACTS.TEST_NXT
+    ]);
+
+    expect(Object.keys(info).map(key => info[key])).toEqual(
+      expect.arrayContaining([
+        {
+          decimals: 8,
+          name: "Red Pulse Token 3.1.4",
+          symbol: "RPX",
+          totalSupply: 43467000
+        },
+        {
+          decimals: 8,
+          name: "LOCALTOKEN",
+          symbol: "LWTF",
+          totalSupply: 2270000
+        },
+        {
+          decimals: 8,
+          name: "NEX Template",
+          symbol: "NXT",
+          totalSupply: 2508560

```

Comment:
Im not certain that these supply numbers will stay static. Do you know? I will accept a test just checking for number type and maybe larger than a arbitrary number (the best we can do if these numbers flutter but I doubt it will go down).

Reply:
Apparently, it is possible for the Total Supply to decrease. I've adjusted the tests